_includes/navbar.html

```diff
@@ -80,7 +80,26 @@
             <li{% if page.url contains '/roadmap.html' %} class="active"{% endif %}><a href="{{ baseurl_i18n }}/roadmap.html">{{ site.data.i18n[page.language].roadmap }}</a></li>
 
             <!-- Contribute -->
-            <li{% if page.url contains '/how-to-contribute.html' %} class="active"{% endif %}><a href="{{ baseurl_i18n }}/how-to-contribute.html">{{ site.data.i18n[page.language].how_to_contribute }}</a></li>
+            <li{% if page.url contains '/contributing/how-to-contribute.html' %} class="active"{% endif %}><a href="{{ baseurl_i18n }}/contributing/how-to-contribute.html">{{ site.data.i18n[page.language].how_to_contribute }}</a></li>
```

Comment:
Menu behavior for "How to Contribute" is not consistent as for "Documentation"

Reply:
This is intentional, and I don't think it's an issue.
I want the sub-pages of the contribution pages to be visible all the time, so that people can quickly access the different contribution guides.
The documentation drop down doesn't solve that, because it is only visible when the top item is clicked.

Reply:
Yes, I was made aware of that issue as well.
How about changing the "What is Flink?" to use the same sub-menu as with the "How to Contribute" pages?

Reply:
I've changed the What is Flink to also use the sub menu.